mllib/src/test/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifierSuite.scala

```diff
@@ -74,6 +74,7 @@ class MultilayerPerceptronClassifierSuite
       .setMaxIter(100)
       .setSolver("l-bfgs")
     val model = trainer.fit(dataset)
+    MLTestingUtils.checkCopy(model)
```

Comment:
Thanks @hhbyyh , I agree that this is maybe not the best general way to test it, but this is how it is done in every other `Model` test.  Maybe this is ok for now and we could follow up with a discussion on the best way to check "basic" ML functionality?